ocs_ci/helpers/disruption_helpers.py

```diff
@@ -22,9 +23,42 @@ class Disruptions:
     resource_count = 0
     selector = None
     daemon_pid = None
+    cluster_kubeconfig = ""
+    pod_ocp = ocp.OCP(
```

Comment:
Changed the scope of the variable POD and renamed it. Adding this in the class Disruptions will help to have the object based on individual instances of the class Disruptions. Otherwise the same POD object will be reused in all test cases present in a test file. This will make difficulty in setting object attributes of POD based on individual instances of the class Disruptions.

Reply:
Class  scope is not needed. This is used in one function only. So moved to the function.